.github/workflows/widget-tactical-overview.yml

```diff
@@ -75,3 +75,22 @@ jobs:
           major_version: ${{ needs.get-version.outputs.major_version }}
           minor_version: ${{ needs.get-version.outputs.minor_version }}
           stability: ${{ needs.get-version.outputs.stability }}
+
+  check-stable-packages:
+    needs: [get-version]
+    strategy:
+      matrix:
+        include:
+        - image: 'almalinux:8'
+          distrib: el8
+        - image: 'almalinux:9'
+          distrib: el9
+        - image: 'debian:bullseye'
+          distrib: bullseye
```

Comment:
since the matrix are always the same, you could move it inside the reusable workflow `check-pkg-availability.yml`

Reply:
indeed, this is much cleaner and less repetition. Moving the matrix to the reusable workflow .